api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -125,10 +125,18 @@ def publish_cdr(cmd_name, args)
       "--additional-reader-group [reader_email]",
       ->(opts, v) { opts.additional_reader_group = v},
       "Additional Google group to include in the reader ACL.")
+  op.add_option(
+      "--additional-writer-group [writer_email]",
+      ->(opts, v) { opts.additional_writer_group = v},
+      "Additional Google group to include in the reader ACL.")
   op.add_option(
     "--source-cdr-project-override [source-cdr-project]",
     ->(opts, v) { opts.source_cdr_project_override = v},
     "Override for the source cdr project where the source dataset is.")
+  op.add_option(
+    "--ttl [ttl]",
+    ->(opts, v) { opts.ttl = v},
+    "Add default ttl to dataset tables. Given in seconds.")
```

Comment:
We would never want to set a TTL on the mainline CDR tables or the genomics CDR, so I'm a bit uncomfortable putting this ttl flag here. Can we make this more specific? I'd probably just make a separate tool if you're using this to create extraction datasets - it doesn't seem to fit the publish use case, or else have a separate clause here which creates the datasets when publishing (with a set of predefined parameters, or more specific flags, e.g. `--extraction-dataset-ttl`).

Also noting that this will be handled on a  different cadence than the normal CDR publish - this is effectively needed once per perimeter that we create.